File: dataset/human/python/file_8405.py
if __name__ == "__main__":
    k, n = map(int, input().split())
    a = list(map(int, input().split()))

    for i in range(n):
        a.append(a[i] + k)

    ans = 2 * k

    for i in range(n):
        tmp = a[i+n-1] - a[i]
        if tmp < ans:
            ans = tmp

    print(ans)

File: dataset/human/c/file_8405.c
#include <stdio.h>
int main (){
    int n,k;
    char s[11];
    char tmp;
    int i=0;

    scanf("%d",&n);
    scanf("%s",s);
    scanf("%d",&k);
    tmp = s[k-1];
    for(i=0; i <n ; i++){
        if (s[i] != tmp) {
            s[i] = '*';
        }
        printf("%c",s[i]);
    }
    return 0;
}
